modules/plugin/shapefile/src/test/java/org/geotools/data/shapefile/ShapefileDataStoreTest.java

```diff
@@ -520,6 +520,29 @@ public void testFidFilter() throws Exception {
         ds.dispose();
     }
     
+    @Test
+    public void testGeometryFilter() throws Exception {
+        File shpFile = copyShapefiles(STREAM);
+        URL url = shpFile.toURI().toURL();
+        ShapefileDataStore ds = new ShapefileDataStore(url);
+        SimpleFeatureSource featureSource = ds.getFeatureSource();
+        SimpleFeatureCollection features;// = featureSource.getFeatures();
+        
+
+        
+        GeometryFactory geometryFactory = new GeometryFactory();
+        Coordinate coordinate = new Coordinate(-99.0,38.0);
+        Point p = geometryFactory.createPoint(coordinate);
+        FilterFactory2 ff = CommonFactoryFinder.getFilterFactory2(null);
+        final Filter testFilter = ff.intersects(ff.literal(p), ff.property("the_geom"));
+        System.out.println(testFilter);
```

Comment:
No system.out in tests please (I use the too, but try to remember to comment them out before committing)